src/content/docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration.mdx

```diff
@@ -20,56 +20,53 @@ redirects:
 freshnessValidatedDate: never
 ---
 
-import kubernetesClusterExplorer from 'images/kubernetes_screenshot-full_cluster-explorer.gif'
+import kubernetesClusterExplorer from 'images/kubernetes_screenshot-crop_cluster-explorer.webp'
 
 import kubernetesDashboard from 'images/kubernetes_screenshot-full_dashboard.webp'
 
 import kubernetesEvents from 'images/kubernetes_screenshot-full_events.gif'
 
-Kubernetes is an open-source system for automating deployment, scaling, and management of containerized applications. The New Relic Kubernetes monitoring integration gives you visibility into your Kubernetes clusters and workloads in minutes, whether your clusters are hosted on-premises or in the cloud.
 
-Thanks to our [cluster explorer](/docs/integrations/kubernetes-integration/cluster-explorer/kubernetes-cluster-explorer), you can cut through layers of complexity to see how your cluster is performing, from the heights of the control plane down to applications running on a single pod.
+Kubernetes is an open-source system for automating deployment, scaling, and management of containerized applications. The New Relic Kubernetes monitoring integration gives you a quick and easy way to see what's going on with your Kubernetes clusters and workloads, whether they're hosted on-premises or in the cloud.
 
-<img
-title="New Relic - Kubernetes Cluster Explorer"
-alt="New Relic - Kubernetes Cluster Explorer"
-src={kubernetesClusterExplorer}
-/>
+Once you've got the [Kubernetes integration up and running](/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure/), you can start monitoring all the services running on Kubernetes in one place with to our [Kubernetes out-of-the-box experiences](/docs/integrations/kubernetes-integration/cluster-explorer/kubernetes-cluster-explorer). It'll show you everything from the heights of the control plane down to applications running on a single pod, so you can get to the heart of how your cluster is performing.
```

Comment:
I have gotten feedback from the field that we link to other docs too frequently- and I agree! I think you should remove these doc links and add them as a "What's Next" at the end.

Reply:
I already have a `Choose your next step` section. This is a description of what you can do with our platform because this is the introduction page.